src/mpi/coll/iallgather/iallgather_tsp_recexch_algos.h

```diff
@@ -237,7 +237,7 @@ int MPIR_TSP_Iallgather_sched_intra_recexch(const void *sendbuf, int sendcount,
     size_t recv_type_size, recv_extent;
     MPI_Aint recv_lb, true_extent;
     int step1_sendto = -1, step2_nphases, step1_nrecvs, p_of_k, T;
-    int dtcopy_id, n_invtcs = 0, invtx;
+    int dtcopy_id, n_invtcs = 0, invtx[1];
```

Comment:
Not crazy about this fix, to be honest. it looks like this gets passed all the way down to `utarray` for concatenation? This might be one I'm willing to ignore in coverity.

Reply:
I don't like it either. Yes, this goes down all the way to utarray but that is not a problem. I will remove this commit.